src/main/java/com/taskadapter/redmineapi/internal/RedmineJSONParser.java

```diff
@@ -357,6 +359,19 @@ public static CustomField parseCustomField(JSONObject content)
 
 		return result;
 	}
+	public static List<CustomField> parseCustomFields(JSONArray content)
+			throws JSONException {
+		final List<CustomField> results = new ArrayList<>();
+		for(int i = 0; i < content.length(); i++)
+		{
+			JSONObject object = content.getJSONObject(i);
+			results.add(CustomFieldFactory.create(JsonInput.getInt(object, "id"),
+					JsonInput.getStringOrEmpty(object, "name"),
+					JsonInput.getStringOrEmpty(object, "value")));
+		}
+			
+		return results;
+	}
```

Comment:
did you see how Project element is parsed? 
`  result.addCustomFields(JsonInput.getListOrEmpty(content, "custom_fields", RedmineJSONParser::parseCustomField));`

was there a reason why the same code did not work for you and you had to implement `parseCustomFields` method from scratch?